charts/hedera-mirror-common/templates/test-trigger-rest.yaml

```
@@ -0,0 +1,21 @@
+{{- if .Values.testkube.enabled -}}
```

may need an additional trigger enabled flag

```suggestion
{{- if and .Values.testkube.enabled .Values.testkube.trigger.enabled -}}
```

since we don't want to have both cron schedule test execution and tigger based test execution in the same environment

might well set the labels to match the test pod in values.yaml so it's also flexible.

---

Great, I have added the trigger property and defaulted it to false.